Source/Core/DolphinQt2/Main.cpp

```diff
@@ -105,7 +105,8 @@ int main(int argc, char* argv[])
   UICommon::CreateDirectories();
   UICommon::Init();
   Resources::Init();
-  Settings::Instance().SetDebugModeEnabled(options.is_set("debugger"));
+  if (options.is_set("debugger"))
+    Settings::Instance().SetDebugModeEnabled(options.is_set("debugger"));
```

Comment:
Not sure how `is_set` is implemented, but you could simply pass `true` here instead. Doesn't really matter in the end though, so +/-0 on actually changing it.